samples/ImplicitFlow/AureliaApp/test/unit/open-id/_test-objects.ts

```diff
@@ -0,0 +1,54 @@
+import {
```

Comment:
Please consider removing all the test-related files from this sample. I'm not sure tests are really useful for a sample.